woocommerce/woocommerce-ios
# [Mobile Payments] Update URL to wp-admin

Closes: #5616 


### Description

Update the URL to wp-admin to rely on the store's adminURL.


### Testing instructions

1. Log in to a store with WooCommerce Payments activated but not setup completely. 
2. Navigate to Settings > In-Person Payments
3. Notice the error, and the url in the web view.


---
